python/ray/serve/handle.py

```diff
@@ -138,7 +122,7 @@ def options(self,
             _internal_use_serve_request=self._use_serve_request)
 
     def _remote(self, endpoint_name, handle_options, args,
-                kwargs) -> Coroutine:
+                kwargs) -> concurrent.futures.Future:
```

Comment:
Nice, so in the future we could elect to just return the `Future` object from both the async and sync handles, right? Maybe a small wrapper that makes it directly awaitable.